README.md

```diff
@@ -12,13 +12,41 @@ A Spectro Cloud demo application. This is the database that supports the Hello U
 
 The [Hello Universe](https://github.com/spectrocloud/hello-universe) app includes an [API server](https://github.com/spectrocloud/hello-universe-api) that relies on a Postgres database for storing the number of clicks and other metadata about the clicks such as the browser, os, and timestamp. For your convenience, a Postgres container is available for use.
 
+## Prerequisites
+
+Ensure [Docker Desktop](https://www.docker.com/products/docker-desktop/) is available on your local machine. 
+
+- Use the following command and ensure you receive an output displaying the version number.
+    ```
+    docker version
+    ```
+    
+Alternatively, you can install [Podman](https://podman.io/docs/installation).
+
+- If you are not running a Linux operating system, create and start the Podman Machine in your local environment. Otherwise, skip this step.
```

Comment:
```suggestion
- If you are not using a Linux operating system, create and start the Podman Machine in your local environment. Otherwise, skip this step.
```

😅